docs/visual/faq.md

```diff
@@ -0,0 +1,214 @@
+---
+id: faq
+title: Visual Testing FAQ
+sidebar_label: FAQ
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+## General
+
+### Can I revert a recently accepted snapshot?
+Yes. After accepting a snapshot, you can revert its acceptance before running another build:
+1. Go to the **State Detail** page.
+2. Click on the **Accepted** status dropdown.
+3. Select **Revert**.<br/>
+   <img src={useBaseUrl('img/visual/revertAccepted.png')} alt="Revert accepted screenshot" width="150" />
+
+### How do I show lazy-loaded content in my screenshot?
+You need to trigger the lazy-loaded content so that it loads and displays in your web page. How to do this depends on how the lazy-loaded content is triggered to load.
```

Comment:
Feedback from Turan: Change: "You need to" to "You will need to"... it just sounds softer.

Reply:
done!